spec.bs

```diff
@@ -1728,13 +1743,21 @@ successful [=navigate|navigation=] to a [=top-level traversable=].
   </wpt>
 </div>
 
-<div algorithm="automatic beacon patch">
+<div algorithm="top_navigation_start beacon patch">
+  Modify [[HTML]]'s [=navigate=] algorithm. Add a new step after step 4 that reads:
+  
+  5. [=attempt to send an automatic beacon=] given <var ignore>sourceDocument</var>, <var
```

Comment:
The signature for this algorithm does not take a sourceDocument as the first parameter.